libs/common/src/lib/prompt-lab/loading-dialog/loading-dialog.component.scss

```diff
@@ -0,0 +1,5 @@
+@import '../../../../../pastanaga-angular/projects/pastanaga-angular/src/lib/modal/confirmation-dialog/confirmation-dialog.component';
```

Comment:
maybe importing from `libs/pastanaga-angular/projects/pastanaga-angular/src/lib/modal/confirmation-dialog/confirmation-dialog.component` would be a bit better

Reply:
yes, it has been changed automatically by my IDE when I moved the component 🙄